solutions/swb-app/src/sshKeys/createSshKeyRequest.ts

```diff
@@ -2,13 +2,13 @@
  *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *  SPDX-License-Identifier: Apache-2.0
  */
-import { z } from 'zod';
+import { z } from '@aws/workbench-core-base';
 
 // eslint-disable-next-line @rushstack/typedef-var
 export const CreateSshKeyRequestParser = z
   .object({
-    projectId: z.string(),
-    userId: z.string()
+    projectId: z.string().projId().required(),
+    userId: z.string().required()
```

Comment:
I'd recommend setting the entire object as required if you are requiring all values

Reply:
required is actually setting the length common requirements as well (ie. not just exist nor just nonempty). But it is a good point that there requests do need everything so the entire zod object should be required too!